conversation/frontend/src/components/MessageActionDropDown/hooks/useMessageActionDropDown.tsx

```diff
@@ -151,17 +167,34 @@ export function useMessageActionDropDown(message: Message, actions?: string[]) {
   };
 
   const handleSendClick = () => {
-    sendDraftQuery.mutate({
-      draftId: message.id,
-      payload: {
-        subject: message.subject,
-        body: message.body,
-        to: recipientToIds(message.to),
-        cc: recipientToIds(message.cc),
-        cci: message.cci ? recipientToIds(message.cci) : undefined,
+    sendDraftQuery.mutate(
+      {
+        draftId: message.id,
+        payload: {
+          subject: message.subject,
+          body: message.body,
+          to: recipientToIds(message.to),
+          cc: recipientToIds(message.cc),
+          cci: message.cci ? recipientToIds(message.cci) : undefined,
+        },
       },
-    });
-    navigate(`/inbox`);
+      {
+        onSuccess: (response) => {
+          if (response.inactive.length > 0 || response.undelivered.length > 0) {
+            if (response.inactive.length > 0) {
+              setInactiveUsers(response.inactive);
+              setOpenedInactiveUsersModal(true);
```

Comment:
```suggestion
              setOpenedInactiveUsersModal(response.inactive);
```
Autant setter directement la valeur à afficher, plutôt qu'un booléen sans valeur ajoutée.